dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -309,7 +309,7 @@
             <Color x:Key="CardBackgroundFillColorDefault">#B3FFFFFF</Color>
             <Color x:Key="CardBackgroundFillColorSecondary">#80F6F6F6</Color>
 
-            <Color x:Key="SmokeFillColorDefault">#4D000000</Color>
+            <Color x:Key="SmokeFillColorDefault">#4DFFFFFF</Color>
```

Comment:
I thought the Light Theme's Smoke colour was changed to be dark to prevent the applications looking like they have crashed.  Why has this change been reverted?

Reply:
Hey Martin. The design team decided it feels better for light theme. It works better with the theme design language system.